components/Layout.jsx

```diff
@@ -1,8 +1,30 @@
-import { Box, Container } from '@mui/material';
+import { Menu as MenuIcon } from '@mui/icons-material';
```

Comment:
```suggestion
import MenuIcon from '@mui/icons-material/MenuIcon';
```

this will allow better bundling and also increase the development performance.
See: https://mui.com/material-ui/guides/minimizing-bundle-size/#bundle-size-matters